icelake/examples/read_iceberg_table.rs

```diff
@@ -1,13 +1,13 @@
 use std::env;
 
 use anyhow::Result;
-use icelake::Table;
+use icelake::catalog::FileSystemCatalog;
```

Comment:
Do we have better names for `FileSystemCatalog`? File system always refer to POSIX fs or HDFS.

Icelake shoule be able to work on any storage services. How about `StorageCatalog`?

Reply:
LGTM